custom/enikshay/case_utils.py

```diff
@@ -35,12 +35,11 @@ def get_parent_of_case(domain, case_id, parent_case_type):
 
```

Comment:
> There seems to be an issue where the case_type is not correct in indices
This sounds like it could be a fairly serious bug. Have you been able to reliably reproduce this wrong case_type on indicies issue? If so, we might want to send over to interrupt to investigate further.